docker-for-windows/release-notes.md

```diff
@@ -82,6 +82,27 @@ Docker Desktop Community 2.1.0.0 contains the following experimental features:
 * When two services have a common exposed port, Docker Desktop exposes the available ports for the second service.
 * Fixed the UI lock when changing the Kubernetes state.
 
+### Known issue
```

Comment:
Make it plural? `issues`?

Reply:
Made it singular as we only have one known issue. I think it reads better with the plural. I've updated the heading.